tests/test_template_message.py

```diff
@@ -125,6 +126,29 @@ def test_cc_bcc(tmp_path):
     assert "Secret" not in plaintext
 
 
+def are_strings_equal(s_1, s_2):
+    """Compare strings ignoring trailing whitespace."""
+    s_1 = s_1.strip() if s_1 else ''
+    s_2 = s_2.strip() if s_2 else ''
+    return s_1 == s_2
```

Comment:
```suggestion
    return s_1.strip() == s_2.strip()
```

Reply:
I don't think this will work. I ran into an issue where `s_1` was `None` in some cases.